executor/write_test.go

```diff
@@ -4291,3 +4291,37 @@ func TestHandleColumnWithOnUpdateCurrentTimestamp(t *testing.T) {
 	tk.MustExec("update t force index(primary) set b = 10 where a = '2023-06-11 10:00:00'")
 	tk.MustExec("admin check table t")
 }
+
+func TestFix56408(t *testing.T) {
```

Comment:
@fzzf678 PTAL. I added the test.

It adds two testCase to cover the case that a duplicated rows is loaded in a new query.